_posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "Book Review - Super Pumped: The Battle for Uber, Mike Isaac"
+date: 2022-09-12
+published: true
+---
+
+[This book](https://www.amazon.com/Super-Pumped-Battle-Mike-Isaac-ebook/dp/B07PZ2B85Y/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=1663623752&sr=8-1) was a total page turner. It has everything you'd want from a good story 
+
+1. famous, familiar characters like Travis Kalanick, Anthony Levandowski, Bill Gurley, Larry Page, Susan Fowler, Masayoshi San
+1. a series of incredible events - both good and bad
+1. a behind-the-curtains peek into a familar product, and a hypergrowth tech company.
```

Comment:
It looks like the list of compelling points was originally a sentence that got split up.  This changes the punctuation when in list form.  I'm don't think you need a comma after diverse, and plot.